composer.json

```diff
@@ -7,41 +7,42 @@
     "php": "~8.2.0 || ~8.3.0"
   },
   "conflict": {
-    "pimcore/admin-ui-classic-bundle": "<1.6 || > 1.6.99",
+    "pimcore/admin-ui-classic-bundle": "<1.7 || > 1.7.99",
     "pimcore/admin-ui-classic-light-theme-bundle": "<1.0 || > 1.0.99",
-    "pimcore/asset-metadata-class-definitions": "<2.1 || > 2.1.99",
+    "pimcore/asset-metadata-class-definitions": "<2.2 || > 2.2.99",
     "pimcore/backend-power-tools-bundle": "<1.2 || > 1.2.99",
-    "pimcore/copilot-bundle": "<1.2 || > 1.2.99",
-    "pimcore/customer-management-framework-bundle": "<4.1 || > 4.1.99",
-    "pimcore/data-hub": "<1.8 || > 1.8.99",
-    "pimcore/data-hub-file-export": "<2.1 || > 2.1.99",
-    "pimcore/data-hub-productsup": "<2.1 || > 2.1.99",
-    "pimcore/data-hub-simple-rest": "<3.0 || > 3.0.99",
-    "pimcore/data-hub-webhooks": "<1.2 || > 1.2.99",
-    "pimcore/data-importer": "<1.9 || > 1.9.99",
+    "pimcore/copilot-bundle": "<1.3 || > 1.3.99",
+    "pimcore/customer-management-framework-bundle": "<4.2 || > 4.2.99",
+    "pimcore/data-hub": "<1.9 || > 1.9.99",
+    "pimcore/data-hub-file-export": "<2.2 || > 2.2.99",
+    "pimcore/data-hub-productsup": "<2.2 || > 2.2.99",
+    "pimcore/data-hub-simple-rest": "<3.1 || > 3.1.99",
+    "pimcore/data-hub-webhooks": "<1.3 || > 1.3.99",
+    "pimcore/data-importer": "<1.10 || > 1.10.99",
     "pimcore/data-quality-management-bundle": "<1.2 || > 1.2.99",
-    "pimcore/direct-edit": "<2.0 || > 2.0.99",
-    "pimcore/ecommerce-framework-bundle": "<1.2 || > 1.2.99",
+    "pimcore/direct-edit": "<2.1 || > 2.1.99",
+    "pimcore/ecommerce-framework-bundle": "<1.3 || > 1.3.99",
     "pimcore/file-explorer-bundle": "<1.0 || > 1.0.99",
     "pimcore/frontend-permission-toolkit-bundle": "<2.0 || > 2.0.99",
-    "pimcore/generic-data-index-bundle": "<1.2 || > 1.2.99",
+    "pimcore/generic-data-index-bundle": "<1.3 || > 1.3.99",
     "pimcore/google-marketing-bundle": "<1.1 || > 1.1.99",
-    "pimcore/headless-documents": "<2.2 || > 2.2.99",
-    "pimcore/newsletter-bundle": "<1.1 || > 1.1.99",
-    "pimcore/object-merger": "<4.0 || > 4.0.99",
-    "pimcore/openid-connect": "<1.1 || > 1.1.99",
-    "pimcore/output-data-config-toolkit-bundle": "<5.1 || > 5.1.99",
+    "pimcore/headless-documents": "<2.3 || > 2.3.99",
+    "pimcore/newsletter-bundle": "<1.2 || > 1.2.99",
+    "pimcore/object-merger": "<4.1 || > 4.1.99",
+    "pimcore/openid-connect": "<1.2 || > 1.2.99",
+    "pimcore/output-data-config-toolkit-bundle": "<5.2 || > 5.2.99",
     "pimcore/personalization-bundle": "<1.1 || > 1.1.99",
-    "pimcore/perspective-editor": "<1.7 || > 1.7.99",
-    "pimcore/pimcore": "<11.4 || > 11.4.99",
-    "pimcore/portal-engine": "<4.1 || > 4.1.99",
+    "pimcore/perspective-editor": "<1.8 || > 1.8.99",
+    "pimcore/pimcore": "<11.5 || > 11.5.99",
+    "pimcore/portal-engine": "<4.2 || > 4.2.99",
+    "pimcore/quill-bundle": "<1.0 || > 1.0.99",
     "pimcore/static-resolver-bundle": "<1.4 || > 1.4.99",
```

Comment:
Shouldn't `v2` be allowed here?

Reply:
We decided to start with static-resolver-bundle v2 in 2025.1. 

But just our of interest, do you know a possibility to create a conflict statement that allows 1.4 and 2.0?

Reply:
What about `<1.4 || >=1.5,<2.0 || >=2.1`? 
I haven't tested it, though — it's just a guess.